package.json

```diff
@@ -16,17 +16,17 @@
     "build": "yarn builder tag:component",
     "build:all": "run-s build:preview build:site migrate:preview",
     "build:preview": "nx build storybook",
-    "build:site": "yarn builder tag:component,ui-icons && gulp buildDocs -LLL",
+    "build:site": "nx build docs",
     "builder": "nx run-many --target build --projects",
     "cache:clean": "nx reset",
-    "ci": "yarn build:all",
-    "ci:storybook": "nx run storybook:build:ci",
+    "ci": "cross-env NODE_ENV=production yarn build:all",
```

Comment:
Setting the NODE_ENV manually to production/development in the package.json influences whether the builder creates map assets.